students/etwum/lesson03/mailroom.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+if __name__ == "__main__":
+    print()
+
+# headers used in table
+lstHeader = [["Donor Name", "| Total Donation(s)", "| # of Donations", "| Avg Donation"]]
+
+# initial list of donors
+lstDonors = [["Lionel Messi", 1000000.00, 5, 20000.00],["Thierry Henry", 500, 1, 500], ["Michael Jordan", 45000, 3, 15000],
```

Comment:
`lstDonors` and `lstHeader` should be snake_case -> `lst_donors` and `lst_header` (something like `header_list` and `donors_list` would be even better)